debian/control

```diff
@@ -0,0 +1,15 @@
+Source: ondemand
+Section: unknown
+Priority: optional
+Maintainer: root <>
+Build-Depends: debhelper (>=11~), ruby
+Standards-Version: 4.1.4
+Homepage: <insert the upstream URL, if relevant>
+
+Package: ondemand
+Architecture: all
+Multi-Arch: foreign
+Depends: ${misc:Depends}, ${ruby:Depends}
```

Comment:
I think need more than `misc` and `ruby`.  The dependency list for RPMs is quite long:

```
# rpm -q --requires ondemand
git
sudo
lsof
cronie
wget
curl
make
rsync
file
libxml2
libxslt
zlib
lua-posix
ondemand-apache >= 2.0-1
ondemand-apache < 3.0
ondemand-apache < 2.1
ondemand-nginx = 1.18.0
ondemand-passenger = 6.0.7
ondemand-ruby >= 2.0-1
ondemand-ruby < 3.0
ondemand-ruby < 2.1
ondemand-python >= 2.0-1
ondemand-python < 3.0
ondemand-python < 2.1
ondemand-nodejs >= 2.0-1
ondemand-nodejs < 3.0
ondemand-nodejs < 2.1
ondemand-runtime >= 2.0-1
ondemand-runtime < 3.0
ondemand-runtime < 2.1
ondemand-gems-2.0.10
```

The `ondemand-*` packages depend on things like `nodejs` and `httpd` or `ruby`.  Also the pattern of putting gems into `ondemand-gems` separate package might be worth looking into since it's what we do with RPMs to ensure apps built against one version of ondemand will continue to work after ondemand updates. If the gems installed for deb packages are not handled the same then updates are going to be extremely painful for a sites using the deb packages.